lib/datastore/request.js

```diff
@@ -755,22 +757,23 @@ DatastoreRequest.prototype.makeReq_ = function(method, body, callback) {
   }
 
   if (method === 'lookup' && this.id) {
-    body.read_options = body.read_options || {};
-    body.read_options.transaction = this.id;
+    body.readOptions = body.readOptions || {};
+    body.readOptions.transaction = this.id;
   }
 
   var pbKey = method[0].toUpperCase() + method.substr(1);
-  var pbRequest = new pb[pbKey + 'Request'](body).toBuffer();
-  var pbResponse = pb[pbKey + 'Response'];
+  var pbRequest = datastoreProto[pbKey + 'Request'].encode(body).toBuffer();
+  var pbResponse = datastoreProto[pbKey + 'Response'];
 
   var reqOpts = {
     method: 'POST',
-    uri: format('{apiEndpoint}/{path}/{projectId}/{method}', {
+    uri: format('{apiEndpoint}/v1beta3/projects/{projectId}:{method}', {
```

Comment:
@pcostell caught the URL format change here :) How will that affect local datastore servers? We accept an `apiEndpoint` option that is usually "localhost:8080" or something similar I believe. Will the path following the host:port match the v1beta3 format?